shesmu-server-ui/src/io.ts

```diff
@@ -334,13 +336,17 @@ export function refreshableSvg<I>(
  *
  * @param input the request to make
  * @param model the model that will deal with the response
+ * @param modal display a modal dialog to lock out the whole UI
  */
 export function requestModel(
   input: RequestInfo,
-  model: StatefulModel<Promise<Response>>
+  model: StatefulModel<Promise<Response>>,
+  modal: boolean
```

Comment:
thank god for typescript or this would seem Perilous

Reply:
I hate it, but I don't have better words.